src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostOptions.cs

```diff
@@ -29,5 +29,22 @@ internal static class RemoteHostOptions
 
         [ExportOption]
         public static readonly Option<bool> RemoteHostTest = new Option<bool>(nameof(InternalFeatureOnOffOptions), nameof(RemoteHostTest), defaultValue: false);
+
+        public const int CrashVS = 0;
+        public const int ShowInfoBarAndRunFeaturesInProcIfPossible = 1;
+        public const int ShowInfoBarAndDisableAllFeatures = 2;
+
+        // Define VS behavior on unintentional OOP shutdown such as user killing OOP process
+        //
+        // CrashVS: 
+        //      crash VS
+        // ShowInfoBarAndRunFeaturesInProcIfPossible: [default]
+        //      show info bar and let VS keep running. some features might keep running inproc based on how they are implemented.
```

Comment:
I don't see how this even works.  Features like Add-Nuget-import likely won't even work if they switch which process they're running in.